README.md

```diff
@@ -212,17 +212,13 @@ Alacritty discussion can be found in `#alacritty` on freenode.
 
 ## Wayland
 
-Wayland support is available, but not everything works as expected. Many people
-have found a better experience using XWayland which can be achieved by
-launching Alacritty with the `WAYLAND_DISPLAY` environment variable cleared:
+Wayland is used by default on systems that support it. Using XWayland may
+circumvent wayland specific issues and can be enabled through:
```

Comment:
```suggestion
circumvent Wayland specific issues and can be enabled through:
```

Super tiny nitpick, but I think we should be consistent here and always capitalize Wayland.